tox.ini

```diff
@@ -84,8 +89,24 @@ commands =
 
 [testenv:flake8]
 basepython = python2.7
-deps = flake8
-commands = flake8 --doctests src tests setup.py
+deps =
+    flake8
+    flake8-html
+    flake8-coding
```

Comment:
This only seems to be needed for Python 2 as in Python 3 the default encoding is utf-8.
Since Python 2.7 (?) a module does not compile if it contains non-ascii characters and the coding is not set.